health-connect/HealthConnectSample/app/build.gradle

```diff
@@ -20,12 +20,12 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk 34
```

Comment:
Suggest upgrading the Android Gradle Plugin if bumping this to 34. (My AS complains the existing version is not compatible with 34.)

Reply:
Good point - done!